.github/scripts/toolchain/create-gcc-summary.sh

```diff
@@ -47,7 +47,7 @@ TOTAL_UNSUPPORTED_TESTS=0
 TOTAL_DEJAGNU_ERRORS=0
 
 set +x # echo off
-for SUM_FILE in "$ARTIFACT_PATH/gcc-tests-$TAG/"*.sum; do
+for FILE in `find $DIR -path '*.sum'`; do
```

Comment:
Why is the `find` needed? Is it for some usecase outside CI?

Reply:
To collect results directly from GCC souce code where `*.sum` files are in different subfolders after running `make check`.